projects/storefrontapp-e2e-cypress/cypress/e2e/b2b/regression/my-company/units.e2e.cy.ts

```diff
@@ -25,7 +25,7 @@ describe('A11y - Units List Keyboard Controls', () => {
     cy.focused().type('{downArrow}');
     cy.focused().should('have.id', 'Rustic Services');
     cy.focused().type('{downArrow}');
-    cy.focused().should('have.id', 'Rustic Services');
+    cy.focused().should('have.id', 'unit-kris1');
```

Comment:
`unit-kris1` value has been added by @Platonn when testing Organization feature, but as far as I remember, only to our dev environment. Please verify if test fails against CI environment

Reply:
Verified on CI, and this part works. There are no units from Kris :)